### PR TITLE
ported gcovr from pypi

### PR DESCRIPTION
**Pull request by mjscosta**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

`@conda-forge/help-python

### CODE REVIEW

File: recipes/gcovr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - gcovr
+
+about:
+  home: http://gcovr.com
+  license: BSD
+  license_family: BSD
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/gcovr/meta.yaml]: Please add the ``license_file``

File: recipes/gcovr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - gcovr
+
+about:
+  home: http://gcovr.com
+  license: BSD
+  license_family: BSD
+  license_file: 
+  summary: A Python script for summarizing gcov data.
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/gcovr/meta.yaml]: Please fill these ``doc`` sections or remove it

File: recipes/gcovr/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/gcovr/meta.yaml]: Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=license_file#packaging-the-licence-manually

**Comment by tovrstra** [comment]
Just so you know. I made a similar recipe here #8062 Main difference being the source in my recipe is the tarbal from github instead of pypi.

**Comment by tovrstra** [comment]
@mjscosta I would be happy to add you as maintainer to the gcovr-feedstock that will soon be created. My apologies for the overlap.

**Comment by mjscosta** [comment]
@tovrstra great then, I've just used conda skeleton for porting the package.

**Comment by mjscosta** [comment]
> @mjscosta I would be happy to add you as maintainer to the gcovr-feedstock that will soon be created. My apologies for the overlap.

@tovrstra if you wish please add me also as a mantainer.

**Comment by tovrstra** [comment]
Apparently, we are both "members" in terms of github terminology, see https://github.com/orgs/conda-forge/teams/gcovr/members I can add you to https://github.com/conda-forge/cpplint-feedstock and https://github.com/conda-forge/cppcheck-feedstock. It would be good to increase the bus factor.

I'll certainly check out your cookiecutter-cpp. I'm trying to get something similar to work, possibly with a somewhat different approach, but there is a lot of overlap. I'm currently trying things out on the following repo: https://github.com/theochem/cellcutoff. The main challenge for me is to make it also work on OSX, which is already a pain. I have not even dared to think about making it work on Windows, just to maintain sanity. :)

**Pull request closed without merging**